recipes/dirent/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class DirEntConan(ConanFile):
+    name = "dirent"
+    description = "Dirent is a C/C++ programming interface that allows programmers to retrieve information about " \
+                  "files and directories under Linux/UNIX"
+    topics = ("conan", "dirent", "directory", "file system")
+    homepage = "https://github.com/tronkko/dirent"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    settings = "compiler"
+    no_copy_source = True
+
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        if self.settings.compiler != "Visual Studio":
+            raise ConanInvalidConfiguration("only Visual Studio builds are supported")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("dirent-{}".format(self.version), self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
```

Comment:
```suggestion
        self.copy(pattern="LICENSE", dst="licenses")
```

Reply:
I changed it to `self.copy(pattern="LICENSE", src=self._source_subfolder, dst="licenses")`